.github/workflows/publish.yml

```diff
@@ -0,0 +1,86 @@
+name: Publish package to npmjs
+
+# This workflow runs when code is pushed to `main` (i.e: when a pull request is merged)
+on:
+    push:
+        branches: [main]
+
+# Ensure that only one instance of this workflow executes at a time.
+# If multiple PRs are merged in quick succession, there will only ever be one publish workflow running and one pending.
+concurrency: ${{ github.workflow }}
+
+jobs:
+    version:
+        runs-on: ubuntu-latest
+
+        # OSBotify will update the version on `main`, so this check is important to prevent an infinite loop
+        if: ${{ github.actor != 'OSBotify' }}
+
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                ref: main
+
+            - name: Decrypt & Import OSBotify GPG key
+              run: |
+                cd .github
+                gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+                gpg --import OSBotify-private-key.asc
+              env:
+                LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+            - name: Set up git for OSBotify
+              run: |
+                git config --global user.signingkey 367811D53E34168C
+                git config --global commit.gpgsign true
+                git config --global user.name OSBotify
+                git config --global user.email infra+osbotify@expensify.com
+            - uses: actions/setup-node@v3
+              with:
+                node-version: '16.x'
+                registry-url: 'https://registry.npmjs.org'
+
+            - name: Generate branch name
+              run: echo "BRANCH_NAME=OSBotify-bump-version-$(uuidgen)" >> $GITHUB_ENV
+
+            - name: Create branch for version-bump pull request
+              run: git checkout -b ${{ env.BRANCH_NAME }}
+
+            - name: Install npm packages
+              run: npm ci
+
+            - name: Update npm version
+              run: npm version patch
+
+            - name: Set new version in GitHub ENV
+              run: echo "NEW_VERSION=$(jq '.version' package.json)" >> $GITHUB_ENV
+
+            - name: Push branch and publish tags
+              run: git push --set-upstream origin ${{ env.BRANCH_NAME }} && git push --tags
+
+            - name: Create pull request
+              run: |
+                gh pr create \
```

Comment:
Let's not use automated / self-merged PRs. Too noisy. Instead, let's create an issue to update `OSBotify`'s permissions in this repo to match his permissions in E/App, react-native-onyx, react-native-live-markdown, and other repos, and then push the version bump directly to main w/o a pull request.

Reply:
@justinpersaud can we update OSBotify's permission as described above? Also happy to create a separate issue if you'd like

Reply:
I'll do it now. Can you create an issue and assign me too? I have to edit the branch protections on the repo and it's going to create a GH for someone to investigate why I'm doing that.

Reply:
added it in

Reply:
Created here - https://github.com/Expensify/Expensify/issues/423724